### PR TITLE
DOC: Google cloud compute documentation

### PR DESCRIPTION
**Pull request by DungLai**
Add documentation for running google cloud compute

### CODE REVIEW

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -326,4 +326,179 @@ Thats it, you are now serving a Surround pipeline! Now you could potentially use
...
 
 .. note:: Since this project was generated with a web runner, the default mode is ``web``, to run the pipeline
-        using the ``FileSystemRunner`` instead, use the command ``$ surround run batch`` or ``$ surround run train``.
\ No newline at end of file
+        using the ``FileSystemRunner`` instead, use the command ``$ surround run batch`` or ``$ surround run train``.
+
+Serving your first pipeline via Google Cloud Compute
+********************************************
+
+**Introduction**
```
Comment (zacbrannelly) [inline docs/source/getting-started.rst]: Use headings instead of just making them bold
e.g. 
```
Introduction
^^^^^^^^^^^^
```

Comment (DungLai) [reply docs/source/getting-started.rst]: I've fixed that ! Thanks Zac

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -326,4 +326,179 @@ Thats it, you are now serving a Surround pipeline! Now you could potentially use
 application.
 
 .. note:: Since this project was generated with a web runner, the default mode is ``web``, to run the pipeline
-        using the ``FileSystemRunner`` instead, use the command ``$ surround run batch`` or ``$ surround run train``.
\ No newline at end of file
+        using the ``FileSystemRunner`` instead, use the command ``$ surround run batch`` or ``$ surround run train``.
+
+Serving your first pipeline via Google Cloud Compute
```
[the remark this answers is not in the document]
Comment (DungLai) [reply docs/source/getting-started.rst]: done, I've created a new rst file called `google-cloud-guide.rst` and put it in the index

**Comment by DungLai** [comment]
This PR is now ready for review

**Comment by AkshatBajaj** [comment]
Merging this to an intermediate branch as it is not ready to be merged into master. We can come back to this once we have more clarity about the direction of the project

**Comment by AkshatBajaj** [comment]
The tests are failing because we have divided that into cli and library and has nothing to do with this PR

**Pull request merged**